openscap_daemon/evaluation_spec.py

```diff
@@ -349,6 +349,16 @@ def get_cpe_ids(self, config):
 
         return cpe_ids
 
+    def select_profile_by_suffix(self, ssg_sds, profile_suffix):
+        profiles = oscap_helpers.get_profile_choices_for_input(ssg_sds, None)
+        for p in profiles:
+            if p.endswith(profile_suffix):
+                self.profile_id = p
+                break
```

Comment:
I think that we don't have multiple profiles with same ID in SSG. But it still makes sense to check for multiple matches. Thanks for suggestion. It will handle if user types some very short suffix, maybe a single character. I'll do that.